tests/test_form_field.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 from django import forms
-from pytest_lazyfixture import lazy_fixture
+from pytest_lazy_fixtures import lf
```

Comment:
```suggestion
from pytest_lazy_fixtures import lf as lazy_fixture
```
and then revert all the remaining changes in this file.